short_course/thirty_minute_lab_tutorial.ipynb

```diff
@@ -2747,7 +2752,21 @@
     }
    ],
    "source": [
-    "regridded_precip_field = precip_field.regrids({'longitude': lon, 'latitude': lat}, method='conservative')\n",
+    "# Create a new Domain\n",
+    "domain = cf.Domain()\n",
+    "\n",
+    "# Create and set the DomainAxis objects\n",
+    "domain_axis_longitude = domain.set_construct(cf.DomainAxis(lon.size))\n",
+    "domain_axis_latitude = domain.set_construct(cf.DomainAxis(lat.size))\n",
+    "\n",
+    "# Set the DimensionCoordinate objects to the domain\n",
+    "domain.set_construct(lon, axes=[domain_axis_longitude])\n",
+    "domain.set_construct(lat, axes=[domain_axis_latitude])\n",
+    "\n",
+    "# Perform conservative regridding\n",
+    "regridded_precip_field = precip_field.regrids(domain, method='conservative')\n",
```

Comment:
Hi Ankit - I was wondering why you didn't go for this more succinct version:
```suggestion
    "\n",
    "# Perform conservative regridding\n",
    "regridded_precip_field = precip_field.regrids([lat, lon], method='conservative')\n",
```

Reply:
Ah! The creation of the domain was in my head, maybe that's why. Thanks for the suggestion David!